tests/queries/0_stateless/02735_parquet_encoder.sql

```diff
@@ -1,4 +1,4 @@
--- Tags: no-fasttest, no-parallel
```

Comment:
Could you please also add `no-msan`? That build is also [tend to be slow](https://play.clickhouse.com/play?user=play#U0VMRUNUIGNoZWNrX3N0YXJ0X3RpbWUsIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgdGVzdF9zdGF0dXMsIGNoZWNrX3N0YXR1cywgcmVwb3J0X3VybCwgdGVzdF9kdXJhdGlvbl9tcwpGUk9NIGNoZWNrcwpXSEVSRSAxCiAgICBBTkQgcHVsbF9yZXF1ZXN0X251bWJlciA9IDAKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnU0tJUFBFRCcKICAgIEFORCB0ZXN0X3N0YXR1cyA9PSAnT0snCiAgICAtLSBBTkQgY2hlY2tfc3RhdHVzICE9ICdzdWNjZXNzJwogICAgQU5EIGNoZWNrX25hbWUgTElLRSAnJW1zYW4lJwogICAgQU5EIHRlc3RfbmFtZSBpbGlrZSAnJTAyNzM1X3BhcnF1ZXRfZW5jb2RlciUnCk9SREVSIEJZIGNoZWNrX3N0YXJ0X3RpbWUgZGVzYywgY2hlY2tfbmFtZSwgdGVzdF9uYW1l). In the private repo we run msan and s3 storage combination, which also times out very often.